exercises/rail-fence-cipher/example.scala

```diff
@@ -0,0 +1,46 @@
+case class RailFenceCipher(rails: Int) {
```

Comment:
Because I wanted to hold the size value as a member variable. 
I suppose I could have just had size as a variable within encode and decode and then passed the value to the track function. But, I didn't think about it.